### DISCUSSION TITLE
Remove items for status bar

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#2508] [Q&A]
I would like to remove the following two items from the default status bar. (Snap Mode and Scope).  Can you please guide.

![StatusBar](https://user-images.githubusercontent.com/83007711/137481040-0fea888f-f471-42b7-be03-cae2d5a2a3d2.JPG)

Thank you,

Ovais

**Comment by aruniverse** [comment]
Hi, at the moment, you can remove the entire status bar but can't remove individual items from it. We'll add this to our backlog and it'll be available in a few weeks in our 3.0 Viewer. If you can't wait until then, I'd recommend opening a PR to our viewer repo for this support.

**Reply by ovaisa** [reply]
Hi Arun, I can wait few weeks, but I hope it will make it into 3.0 Viewer.

Can you please point me to viewer repo.

**Reply by aruniverse** [reply]
Here's the [viewer monorepo](https://github.com/iTwin/viewer)
The change would need to be made in the [@itwin/viewer-react](https://github.com/iTwin/viewer/tree/master/packages/modules/viewer-react) package, specifically you'd have to update the following files for sure: 
- https://github1s.com/iTwin/viewer/blob/master/packages/modules/viewer-react/src/types.ts 
- https://github1s.com/iTwin/viewer/blob/master/packages/modules/viewer-react/src/components/app-ui/frontstages/DefaultFrontstage.tsx
- https://github1s.com/iTwin/viewer/blob/master/packages/modules/viewer-react/src/components/app-ui/statusbars/AppStatusBar.tsx

If you're interested in contributing the enhancement, I can continue to guide you. 

In 3.0 we are aiming to make the viewer more configurable, so we'll make sure to support this.
